### PR TITLE
ClothesAdded new clothe to the clothemate vendor MK2

### PR DESCRIPTION
**Pull request by enkidienne**
# General Documentation
list of added clothes:

regular (50$):

cowboy boots: lizard
trainers
black suit jacket
gothic coat
gothic suit
White and black sweatshirt
white suit

premium (75$):
Pharoah tunic
### Why is this change good for the game?

moar options = good.
.
.
.

no?
# Changelog

:cl:  
adds more clothes to clothemate vendor's list
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
bro you literally just remade the pr that got rejected and you even have the masterwork lizard boots still in this pr what

**Comment by enkidienne** [comment]
typo in the log of what i changed, and last one was closed due to a feature freeze

**Comment by UselessTheremin** [comment]
ok so some criticisms:

- botanist and mining winter coats don't really have a reason to be put there, they have their own designated machines to be vended from.
- the cyborg mask is already in the autodrobe, no reason for it to also be in here.
- again, i think that the russian gangster suit is already in the autodrobe. why does it need to be in the clothesmate?
- all of the trainers already come from a designated cargo crate to make them a bit more special which i think is currently a good way of doing that
as it stands, there is bloat in these vending machines. there needs to be a few things trimmed in my eyes, and i will probably actually do this in the near future. but for now i honestly think: more does not necessarily equal good. especially when, as i said, bloat is already an apparent problem in this particular vending machine. nobody is really going to use these specific clothing items from the clothesmate, i can almost guarantee you.

**Comment by enkidienne** [comment]
sure, done

File: code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -153,18 +156,28 @@
...
 		            /obj/item/clothing/under/yogs/whitetuxedo = 1,
 		            /obj/item/clothing/under/yogs/whitedress = 1,
 		            /obj/item/clothing/under/yogs/fancysuit = 1,
 		            /obj/item/clothing/glasses/sunglasses/cheap = 2,
 		            /obj/item/clothing/glasses/yogs/threed = 2,
 		            /obj/item/clothing/shoes/sneakers/black = 3,
 		            /obj/item/clothing/shoes/yogs/fire_crocs = 1,
+		            /obj/item/clothing/shoes/winterboots = 2,
+		            /obj/item/clothing/shoes/singery = 1,
+		            /obj/item/clothing/shoes/singerb = 1,t
```
Comment (Bibby0110) [inline code/modules/vending/clothesmate.dm]:
```suggestion
		            /obj/item/clothing/shoes/singerb = 1,
```

Comment (enkidienne) [reply code/modules/vending/clothesmate.dm]: why the "T"?

Comment (enkidienne) [reply code/modules/vending/clothesmate.dm]: ho wait nevermind im stupid

**Review by SomeguyManperson — APPROVED** [review]

**Pull request merged**